Makefile

```diff
@@ -5,20 +5,36 @@ BROKER_IMAGE     ?= $(REGISTRY)/$(ORG)/ansible-service-broker
 BUILD_DIR        = "${GOPATH}/src/github.com/openshift/ansible-service-broker/build"
 PREFIX           ?= /usr/local
 BROKER_CONFIG    ?= $(PWD)/etc/generated_local_development.yaml
+SOURCE_DIRS      = cmd pkg
 SOURCES          := $(shell find . -name '*.go' -not -path "*/vendor/*")
 SVC_ACCT_DIR     := /var/run/secrets/kubernetes.io/serviceaccount
 KUBERNETES_FILES := $(addprefix $(SVC_ACCT_DIR)/,ca.crt token tls.crt tls.key)
 .DEFAULT_GOAL    := build
 
-vendor:
+vendor: ## Install project dependencies
 	@glide install -v
 
-broker: $(SOURCES)
-	go build -i -ldflags="-s -w" ./cmd/broker
+broker: $(SOURCES) ## Build the broker
+	@go build -i -ldflags="-s -w" ./cmd/broker
 
-build: broker
+build: broker ## Build the project
 	@echo > /dev/null
 
+lint: ## Run golint
+	@golint -set_exit_status $(SOURCE_DIRS)
```

Comment:
This doesn't work.

```
[jesusr@speed3 ansible-service-broker{djzager-simple-make}]$ make lint
read cmd: is a directory
read pkg: is a directory
[jesusr@speed3 ansible-service-broker{djzager-simple-make}]$ 
```

If you looked at the travis.sh before golint was run separately:

```
-  # to have lint fail build add -set_exit_status option
-  # to each golint command
-  if [ -n "$(golint $PROJECT_ROOT/cmd/...)" ]; then
-        golint $PROJECT_ROOT/cmd/...
-        exit 1
-  fi
-  if [ -n "$(golint $PROJECT_ROOT/pkg/...)" ]; then
-        golint $PROJECT_ROOT/pkg/...
-        exit 1
-  fi
```

you have to call golint twice once for each package and it MUST contain the `...` after the package name.